tools/tangle-cli/src/commands/msg/submitMsgCommandExecutor.ts

```diff
@@ -1,44 +1,49 @@
-import { Converter, SingleNodeClient, IMessage, IIndexationPayload, INDEXATION_PAYLOAD_TYPE } from "@iota/iota.js";
+import {
```

Comment:
we need a prettier setup aligned with what @obany is doing in other codebases so that there are no discrepancies on prettifying code